### PR TITLE
Added: Auth

### PR DESCRIPTION
**Pull request by hussiantabha**
Now users can log in and signup and can also navigate to the page on which they clicked login

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=hussainTabha
```
Comment (t007rushi) [inline .env:1]: don't push
put in ```.gitignore```

File: src/components/NormalNavbar.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{
+  /* <div className="nav-fixed">
+  <div className="nav-header">
+    <h1 className="nav-logo">AdTube</h1>
+    <span className="nav-subtext">A YT without Ad</span>
+  </div>
+  <div>
+    <button className="btn btn-primary">Login</button>
+  </div>
+</div>; */
```
Comment (t007rushi) [inline src/components/NormalNavbar.js:73]: remove if not needed

File: src/pages/Login.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import Navbar from "../components/Navbar";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useNavigate } from "react-router-dom";
+import "../css/login.css";
+import NormalNavbar from "../components/NormalNavbar";
+
+const Login = () => {
+  let location = useLocation();
+  console.log(location);
```
Comment (t007rushi) [inline src/pages/Login.js]: consoles to remove

File: src/pages/Login.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  const loginFunc = async () => {
+    try {
+      const postData = await fetch("/api/auth/login", {
+        method: "POST",
+        headers: {
+          "Content-type": "application/json",
+        },
+        body: JSON.stringify(loginInput),
+      });
+      console.log(postData);
```
Comment (t007rushi) [inline src/pages/Login.js]: same here

**Review by t007rushi — APPROVED** [review]
There are lot of consoles to remove 
else looks that followed good folder structure
and naming

**Review by mariyasada — APPROVED** [review]
this code looks good to me. liked that you have used toast. just remove all the console.log

**Pull request merged**